packages/fiori/src/Wizard.ts

```diff
@@ -233,6 +234,13 @@ type StepInfo = {
 })
 
 class Wizard extends UI5Element {
+	/**
+	 * Defines how the steps of the <code>ui5-wizard</code> would be visualized.
+	 * @public
+	 */
+	@property({ validator: String, defaultValue: WizardRenderMode.Scroll })
+	renderMode?: WizardRenderMode
```

Comment:
Hi guys, just wanted to comment that `renderMode` sounds very technical (I know it's the same in sapui5), but are any other options also considered?

Some other perspective can be to look at scrolling as a single page design, and the page mode as multiple pages mode.

design, contentDesign: 
- SinglePage (e.g Scrolling)
- MultiplePages (e.g Page)

Reply:
Thank you for the suggestion, we are definitely going to consider this.